av/utils.pxd

```diff
@@ -9,5 +9,7 @@ cdef dict_to_avdict(lib.AVDictionary **dst, dict src, str encoding, str errors)
 cdef object avrational_to_fraction(const lib.AVRational *input)
 cdef object to_avrational(object value, lib.AVRational *input)
 
+cdef object name_to_codec_tag(object name)
```

Comment:
Some questions about types:

- I see codec_tag is declared as `unsigned int` (so length can depend on the platform), not `uint32_t`
- how come name_to_codec_tag returns an object and not an `unsigned int`?

Reply:
Good points. I'll change the codec_tag type to `unsigned int` (although I'm not sure how the tags are supposed to work when `unsigned int` is 2 bytes).
I'll make the other changes as well. I think `str` is a more readable/user friendly type to expose as long as we account for errors.
This is my first time touching Cython, so please forgive any obvious problems.

Reply:
There's nothing obvious about these problems, I didn't spot them at first! Do you have any examples of non-ascii tags used by codecs?

Reply:
I had a look at how the FFmpeg takes tags from the command line, and I *think* it's manipulating them as uint32t:

https://github.com/FFmpeg/FFmpeg/blob/master/fftools/ffmpeg_opt.c#L821-L827

Reply:
Moved it to draft as I'm playing around with using python built-ins instead of adding extra utility functions.

Reply:
Changed it to use the utility functions. The getter shouldn't have any problems since we're converting from the c integer type to a more flexible python int type. Will the setter have issues if we try to set an overflowing integer into a 2 byte unsigned int?

Reply:
I guess it will just raise an OverflowError, but we probably don't have to handle it